src/drop/null-drop.ts

```diff
@@ -22,3 +22,5 @@ export class NullDrop extends Drop implements Comparable {
     return null
   }
 }
+
+export const NULL = new NullDrop()
```

Comment:
nice one! could you also help change other `new NullDop` calls to `NULL`?

Reply:
Sure. I'll do a separate PR.